### DISCUSSION TITLE
Horse menu

### DISCUSSION THREAD

**Discussion by Fr03Ab4l4** [YimMenu/HorseMenu#119] [Q&A] [answered]
how to use the menu and which injector should I use?

**Comment by Rxann** [comment]
I personally recommend the xenos injector and you download the dll from the releases tab, and put that into the injector then boom. click insert on your keyboard once its injected.

**Reply by Fr03Ab4l4** [reply]
thanks man how do i open the menu what is the key

**Reply by Rxann** [reply, marked as answer]
The insert key.
